content/en/docs/Concepts/crds/bundleIndexImage.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Bundle Index Image"
+weight: 3
+---
+
+
+A bundle is an operator packaging construct which contains an operator definition and manifests which ultimately determine how the operator is deployed onto a Kubernetes cluster. Bundles are the preferred mechanism within OLM going forward to package and deploy Operators. Operator developers are encouraged to use the Bundle format over PackageManifests.
+
+To build an Operator index image, we must first have an Operator that is ready to be deployed. To create an Operator, we can use `operator-sdk` to scaffold your Operator and construct your Operator. 
+
+Once your Operator is built, the resultant code is built into a container image. If you use [Operator SDK](https://sdk.operatorframework.io/docs/) to build your Operator, it contains a `Makefile` that you can use to build and push your Operator image. You can use one of the following commands to create and push an image on your Operator registry:
+
+```
+make docker-build docker-push IMG=<registry>/<username>/sample-operator:vX.X.X
+```
+
+### Building the Operator Bundle Image
+
+For an Operator to be deployed it has to be bundled. We can create the Bundle using one of the following commands:
+```
+make bundle IMG=<registry>/<username>/sample-operator:vX.X.X
+```
+
+The `make bundle` command also creates a Dockerfile (bundle.Dockerfile) which is used to build a "bundle image." The bundle image is an [OCI](https://github.com/opencontainers/image-spec) image that essentially holds the generated on-disk bundle manifest and metadata files.`
+
+### Building the Operator Index Image
+
+Another OLM concept is the "index image", which is like a catalog of Operators. This container image acts to serve an application programming interface (API) which describes information about your sample operator and others in the catalog. An index image, based on the Operator Bundle Format, is a containerized snapshot of a catalog. It is an immutable artifact that contains the database of pointers to a set of Operator manifest content. A catalog can reference an index image to source its content for OLM on the cluster. 
+
+#### Creating an index image:
+
+
+1. The index image includes information from your bundle image by running an operator-registry command, opm, as follows:
```

Comment:
Please replace the following wording as follows: 
The index image includes information from your bundle image by running the operator-registry command by using the opm command as follows: